core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -323,7 +323,11 @@ public void onLoad(ItemGroup<? extends Item> parent, String name) throws IOExcep
         builds = buildMixIn.getRunMap();
         triggers().setOwner(this);
         for (Trigger t : triggers()) {
-            t.start(this, Items.currentlyUpdatingByXml());
+            try {
+                t.start(this, Items.currentlyUpdatingByXml());
+            } catch (Exception e) {
+                LOGGER.log(Level.WARNING, "could not start trigger while loading project '" + name + "'", e);
```

Comment:
No way, `getMessage` will probably be useless. (`null` in the case of a typical `NullPointerException`.) You need the stack trace to diagnose anything.